roles/accprovision/README.md

```diff
@@ -0,0 +1,43 @@
+Role Name
```

Comment:
Seems like this is auto-generated placeholder. Should we be putting some high level info here?